### PR TITLE
Updated the installation guide using Ansible  for Navarino release

### PR DESCRIPTION
**Pull request by babu-mani**
What type of PR is this?

documentation

What this PR does / why we need it:
Document update for Navarino release

Which issue(s) this PR fixes:

Fixes 

Test Report Added?
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

Test Report:

### CODE REVIEW

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 1]
```diff
@@ -5,94 +5,47 @@ weight: 40
...
+- SODA installation is tested on `Ubuntu 18.04`& `Ubuntu 20.04`. 
+- Python 3.6 or above should be installed
 
 ```bash
-dpkg -i docker-ce_18.06.1~ce~3-0~ubuntu_amd64.deb
+sudo apt-get update && sudo apt-get install -y git
 ```
----
-####  Install docker-compose:
+- Ensure no ansible & docker installed, OR Lastest ansible and docker tools are installed with versions listed below or later. If ansible & docker is not installed in the OS, script          install_dependencies.sh will install it.
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:22]:
```suggestion
- Ensure no ansible & docker installed, OR Lastest ansible and docker tools are installed with versions listed below or later. If ansible & docker is not installed in the OS, script          `install_dependencies.sh` will install it.
```

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 2]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
 ---
 #### Enable Delfin installation
 delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It covers the resource management of all the backends & other infrastructures under SODA deployment. It also provides alert management and metric data (performance/health) for monitoring and further analysis.
 
-To install Delfin, update the file `group_vars/delfin.yml` and change the value of `enable_delfin` to `true`
+  -Enable following configration  
+    - In file installer/ansible/group_vars/delfin.yml `enable_delfin: true`  
+    - In file installer/ansible/group_vars/srm-toolchain.yml `install_srm_toolchain: true`  
+    - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`  
+    - **[Click here](#configure-delfin-installation) for more config**
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:83]:
```suggestion
- **[Click here](#configure-delfin-installation) for other supported configuration details **
```

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 3]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+To install SODA Projects and enable the different features, variables have to be modified in the respective files as below:
+
+**_Note: Hotpot ,Gelato & Delfin can be used either through dashboard or REST APIs._**
 
 ---
 #### Enable Delfin installation
 delfin (Dolphin in Spanish!), the SODA Infrastructure Manager project is an open source project to provide unified, intelligent and scalable resource management, alert and performance monitoring. It covers the resource management of all the backends & other infrastructures under SODA deployment. It also provides alert management and metric data (performance/health) for monitoring and further analysis.
 
-To install Delfin, update the file `group_vars/delfin.yml` and change the value of `enable_delfin` to `true`
+  -Enable following configration
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:79]:
```suggestion
  -Enable following configuration  
```

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 4]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+---
+
+
+#### Enable Hotpot  
+ Hotpot installs SODA On Premise only.  
+- Enable following configurations
+  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for more config**
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:94]: Change as above

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 5]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for more config**  
+
+#### Enable Gelato
+Gelato installs SODA Multicloud only.
+- Enable following configurations  
+  - In file installer/ansible/group_vars/gelato.yml  update the value  `enable_gelato : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#enable-storage-service-plans-in-multicloud-\(optional\)) for more config**
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:102]: Change as above

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 6]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+#### Enable Gelato
+Gelato installs SODA Multicloud only.
+- Enable following configurations  
+  - In file installer/ansible/group_vars/gelato.yml  update the value  `enable_gelato : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#enable-storage-service-plans-in-multicloud-\(optional\)) for more config** 
 
 ---
+### Install SODA
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:105]:
```suggestion
### Run SODA Installer
```

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 7]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
-`gelato` installs SODA Multicloud only.  
 
-```bash
-# This field indicates which project should be deploy
-# 'hotpot', 'gelato' or 'all'
-deploy_project: all  #all refers to hotpot + gelato
-```
-**_Note: Delfin and other member projects will be added shortly._**
+
+To install SODA Projects and enable the different features, variables have to be modified in the respective files as below:
```
Comment (joseph-v) [inline content/soda-gettingstarted/installation-using-ansible.md:71]:
```suggestion
To install SODA Projects and enable the different features, installer needs to be configured using the respective files as below:
```

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 1 above)
Comment (anvithks) [reply content/soda-gettingstarted/installation-using-ansible.md:22]: If ansible & docker is not installed in the OS -> If ansible & docker **are** not installed in the OS

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 8]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+
+**_Note: Hotpot ,Gelato & Delfin can be used either through dashboard or REST APIs._**
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:73]: Hotpot -> Terra (previously known as Hotpot)
Gelato -> Strato (SODA Multicloud project, previously known as Gelato)

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 9]
```diff
@@ -110,48 +63,87 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+
+#### Enable Hotpot  
+ Hotpot installs SODA On Premise only.  
+- Enable following configurations
+  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for more config**  
+
+#### Enable Gelato
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:96]: Enable Strato.
let's use the project name Strato in the description. Mention the variable as gelato.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 10]
```diff
@@ -339,10 +331,13 @@ delfin_exporter_alertmanager_port: 9093
 --- 
 
 ### Configure SRM Toolchain installation
-
+Delfin produces metrics which can be consumed by any of the exporters that are supported. Currently Delfin supports the Prometheus and Kafka exporters.
+The SRM Toolchain is required to view the metrics and visualization in the SODA Dashboard.\  
 Installing the SRM toolchain will install Prometheus, AlertManager and Grafana versions as per the configuration below and can be changed.
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:337]: These instructions are missing which file needs to be modified. 
This was available earlier. Why is it removed?
_Update the file `ansible/group_vars/srm-toolchain.yml` and change the value of `install_srm_toolchain` to `true`.  
If this value is set to false then the metrics and visualization will not be available using SODA Dashboard._

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the review comments.

**Comment by anvithks** [comment]
Please make the corrections and push a new commit. Seems like a momentary issue with the CI. It should work if you push another commit.

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 10 above)
Comment (babu-mani) [reply content/soda-gettingstarted/installation-using-ansible.md:337]: Previously two blocks were there for SRM toolchain 
1st for definition and 2nd for configuration 
now we refactored and combined in both in one block 
and install_srm_toolchain is available and in the Delfin installation process install_srm_toolchain to true is given

Comment (anvithks) [reply content/soda-gettingstarted/installation-using-ansible.md:337]: > Previously two blocks were there for SRM toolchain 1st for definition and 2nd for configuration now we refactored and combined in both in one block and install_srm_toolchain is available and in the Delfin installation process install_srm_toolchain to true is given

ok

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 11]
```diff
@@ -5,94 +5,46 @@ weight: 40
...
 
 {{% notice info %}}
-**Ubuntu 16.04 support for SODA is deprecated as of SODA Jerba release v1.4.0.** <br />
-**If you have a requirement to install SODA on Ubuntu 16.04, please contact us on slack and we will try to help with the setup.**
-{{% /notice %}}
-
-{{% notice info %}}
-**Ubuntu 20.04 support for SODA is added as experimental in Lamu v1.6.0 release through branch `ubuntu2004-experimental`.** <br />
-**If you have a requirement to install SODA on Ubuntu 20.04, please [refer](https://github.com/sodafoundation/installer/tree/ubuntu2004-experimental) or contact us on slack and we will try to help with the setup.**
+**Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04.** <br />
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:10]: _Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04_  -> **This installation guide is supported on Ubuntu 18.04 & Ubuntu 20.04.**

Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:10]: _Below process of installation will support for Ubuntu 20.04 & Ubuntu 18.04_  -> **This installation guide is supported on Ubuntu 18.04 & Ubuntu 20.04.**

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 12]
```diff
@@ -5,94 +5,46 @@ weight: 40
...
+- Python 3.6 or above should be installed
 
 ```bash
-apt-get update && apt-get install -y git make curl wget libltdl7 libseccomp2 libffi-dev gawk
+sudo apt-get update && sudo apt-get install -y git
 ```
----
-
-#### Install docker:
+- Ensure no ansible & docker installed, OR Lastest ansible and docker tools are installed with versions listed below or later. If ansible & docker are not installed in the OS, script `install_dependencies.sh` will install it.
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:22]: _Please change this one line as below:_
- Ensure Ansible & Docker are not installed
- If Ansible and Docker are installed, ensure that the latest version of Ansible and Docker tools are installed with versions listed below.
- If Ansible & Docker are not installed then use the `install_dependencies.sh` script, as described below, to install them.

Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:22]: Where are the Ansible and Docker versions listed?

**Review by anvithks — CHANGES_REQUESTED** [review]
@babu-mani Please check the review comments.

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 12 above)
Comment (babu-mani) [reply content/soda-gettingstarted/installation-using-ansible.md]: Updated

Comment (babu-mani) [reply content/soda-gettingstarted/installation-using-ansible.md]: Updated

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 11 above)
Comment (babu-mani) [reply content/soda-gettingstarted/installation-using-ansible.md]: updated

Comment (babu-mani) [reply content/soda-gettingstarted/installation-using-ansible.md]: updated

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 13]
```diff
@@ -5,94 +5,53 @@ weight: 40
...
-
-#### Install docker:
-
 
-##### For Ubuntu 18.04
+- Python 3.6 or above should be installed 
+- If Ansible, Docker & Go are not installed in the OS, script `install_dependencies.sh` will install it.
+- If Ansible, Docker & Go are installed ensure that the version as listed below.
+  - Docker: Latest (Verfied on 20.10.21)
+  - Ansible: Latest (Verfied on 5.10.0 )
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:27]: Please specify that this is the Ansible package version. Also mention the ansible-core version that is supported.

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 14]
```diff
@@ -110,48 +69,86 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
-```
+
+#### Enable Terra  
+ Terra installs SODA On Premise only.  
+- Enable following configurations  
+  - In file installer/ansible/hotpot.yml  update the value  `enable_hotpot : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#configure-soda-on-premise-installation) for other supported configuration details**   
+
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:100]: Add a line here ---

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 15]
```diff
@@ -110,48 +69,86 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+#### Enable Strato
+Strato installs SODA Multicloud only.
+- Enable following configurations  
+  - In file installer/ansible/group_vars/gelato.yml  update the value  `enable_gelato : true`.
+  - In file installer/ansible/group_vars/dashboard.yml `enable_dashboard: true`
+  - In file installer/ansible/group_vars/common.yml `host_ip : <User's IP address, eg. 127.0.0.1>`  
+  - **[Click here](#enable-storage-service-plans-in-multicloud) other supported configuration details**
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:107]: Click here **for** other.....

File: content/soda-gettingstarted/installation-using-ansible.md [hunk 16]
```diff
@@ -110,48 +69,86 @@ By default the `host_ip` is set to `127.0.0.1` i.e. localhost.
...
+[verbosity level: -vv < -vvv]
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts -vvv
 ```
----
 
-#### Enable Storage Service Plans in multi-cloud (optional)
+SODA ansible installer supports installation using tags. To install a particular service use the playbook as follows
+```bash
+# This installs only delfin
+sudo -E env "PATH=$PATH" ansible-playbook site.yml -i local.hosts -vvv --tags delfin
```
Comment (anvithks) [inline content/soda-gettingstarted/installation-using-ansible.md:129]: Please check if the `delfin` tag works. @joseph-v Does it work?

**Review by anvithks — CHANGES_REQUESTED** [review]
@babu-mani Overall LGTM. Some comments to fix minor issues.

File: content/soda-gettingstarted/installation-using-ansible.md (hunk 16 above)
Comment (joseph-v) [reply content/soda-gettingstarted/installation-using-ansible.md:129]: > Please check if the `delfin` tag works. @joseph-v Does it work?

Yes

**Review by anvithks — DISMISSED** [review]
LGTM

**Review by joseph-v — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**